scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/utils/ForkerUtils.java

```diff
@@ -48,6 +52,29 @@
     /** System property Key of the fork method */
     public static final String FORK_METHOD_KEY = "pas.launcher.forkas.method";
 
+    // allows to configure the runasme fork method for a workflow task
+    public static final String RUNAS_METHOD_GENERIC_INFO = "RUNAS_METHOD";
+
+    // allows to configure the runasme user for a workflow task
+    public static final String RUNAS_USER_GENERIC_INFO = "RUNAS_USER";
+
+    // allows to configure the runasme user domain for a workflow task
+    public static final String RUNAS_DOMAIN_GENERIC_INFO = "RUNAS_DOMAIN";
```

Comment:
Can you give an example of the RunAS domain?

Reply:
domains are a windows feature. I will add this mention in the comment.